src/test/java/com/mateacademy/lessons15/CustomerDaoImplIntegrationTest.java

```diff
@@ -13,10 +12,14 @@
 public class CustomerDaoImplIntegrationTest {
```

Comment:
It doesn't work correctly. Start full test class. In result, some of test will be "red"